app/assets/stylesheets/application.css.scss

```diff
@@ -8,179 +8,16 @@
  * You're free to add application-wide styles to this file and they'll appear at the top of the
  * compiled file, but it's generally better to create a new file per style scope.
  *
```

Comment:
For consistency with the controllers and views, we should probably call the folder "admin".